addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/discovery/HarmonyHubDiscovery.java

```diff
@@ -52,7 +52,7 @@
 
     private static final int DISCO_PORT = 5224;
 
-    static protected final ScheduledExecutorService scheduler = ThreadPoolManager
+    private static final ScheduledExecutorService SCHEDULER = ThreadPoolManager
```

Comment:
@digitaldan Did we also have any reason for this thread pool here instead of using the existing `scheduler`?